test/integration/scene-stream-access/get-scene-stream-access-handler.spec.ts

```diff
@@ -0,0 +1,297 @@
+      cleanup.trackInsert('scene_stream_access', dbFields)
+    }
+
+    expect(body).toEqual({
+      streaming_url: mockSceneStreamAccess.streaming_url,
+      streaming_key: mockSceneStreamAccess.streaming_key,
+      created_at: Number(mockSceneStreamAccess.created_at),
+      ends_at: Number(mockSceneStreamAccess.created_at) + FOUR_DAYS
+    })
+  })
+
+  it('returns 200 with streaming access when user is an admin', async () => {
+    const { localFetch } = components
+
+    stubComponents.land.hasLandPermission.resolves(false)
+    stubComponents.world.hasWorldOwnerPermission.resolves(false)
+    stubComponents.sceneAdminManager.isAdmin.resolves(true)
+
+    const response = await makeRequest(
+      localFetch,
+      '/scene-stream-access',
+      {
+        method: 'GET',
+        metadata: metadataLand
+      },
+      admin
+    )
+
+    expect(response.status).toBe(200)
+    const body = await response.json()
+
+    if (body) {
+      const { ends_at, ...dbFields } = body
+      cleanup.trackInsert('scene_stream_access', dbFields)
+    }
+
+    expect(body).toEqual({
+      streaming_url: mockSceneStreamAccess.streaming_url,
+      streaming_key: mockSceneStreamAccess.streaming_key,
+      created_at: Number(mockSceneStreamAccess.created_at),
+      ends_at: Number(mockSceneStreamAccess.created_at) + FOUR_DAYS
+    })
+  })
+
+  it('returns 200 with a new streaming access when it does not exist', async () => {
+    const { localFetch } = components
+
+    stubComponents.places.getPlace.resolves({
+      id: anotherPlaceId,
+      positions: ['15,20'],
+      owner: owner.authChain[0].payload
+    } as PlaceAttributes)
+
+    const response = await makeRequest(
+      localFetch,
+      '/scene-stream-access',
+      {
+        method: 'GET',
+        metadata: metadataLand
+      },
+      admin
+    )
+
+    if (response.status === 200) {
+      const body = await response.json()
+      if (body) {
+        const { ends_at, ...dbFields } = body
+        cleanup.trackInsert('scene_stream_access', dbFields)
+      }
+
+      expect(body).toHaveProperty('streaming_url')
+      expect(body).toHaveProperty('streaming_key')
+    }
+  })
+
+  it('returns 200 when user is not owner or admin', async () => {
+    const { localFetch } = components
+
+    stubComponents.land.hasLandPermission.resolves(false)
+    stubComponents.world.hasWorldOwnerPermission.resolves(false)
+    stubComponents.sceneAdminManager.isAdmin.resolves(false)
+    stubComponents.sceneManager.hasPermissionPrivilege.resolves(true)
+
+    const response = await makeRequest(
+      localFetch,
+      '/scene-stream-access',
+      {
+        method: 'GET',
+        metadata: metadataLand
+      },
+      nonOwner
+    )
+
+    expect(response.status).toBe(200)
+    const body = await response.json()
+    expect(body).toHaveProperty('streaming_url')
+    expect(body).toHaveProperty('streaming_key')
+  })
+
+  it('returns 401 if no sceneId in a land request', async () => {
```

Comment:
We should be returning a 400 in those cases, what do you think?
```suggestion
  it('returns 400 if no sceneId in a land request', async () => {
```